deps.edn

```diff
@@ -266,11 +266,11 @@
 
   ;; clojure -M:lib/java11-source
   :lib/java8-source
-  {:extra-deps
+  {:replace-deps
    {openjdk/java-sources {:local/root "/usr/lib/jvm/openjdk-8/lib/src.zip"}}}
 
   :lib/java11-source
-  {:extra-deps
+  {:replace-deps
    {openjdk/java-sources {:local/root "/usr/lib/jvm/openjdk-11/lib/src.zip"}}}
 
```

Comment:
I use the `:lib/java11-source` alias as a development time alias with projects.  If I understand correctly `:replace-deps` would not include any project dependencies and so the project would not work.
This would be the same for `:lib/java-source`.

Reply:
Yes, there are definitely some cases where replacing the deps is wrong and I was too aggressive.  The `:lint/eastwood` task is another example (I'll fix the PR for that one shortly).

But the general principle of switching to `replace-deps` from `extra-deps` is valid: it documents the behavior of the tool (does it run "in-process" with the project or not?), provides a little boost in startup performance and lowers the risk of a dependency clash.

Reply:
Yes, I understand the concept now.  Thanks for raising this.